arches_lingo/src/arches_lingo/api.ts

```diff
@@ -0,0 +1,43 @@
+import arches from "arches";
+import Cookies from "js-cookie";
+
+function getToken() {
+    const token = Cookies.get("csrftoken");
+    if (!token) {
+        throw new Error("Missing csrftoken");
```

Comment:
needs i18n

Reply:
The only value this has for an end user is to copy/paste it into a support ticket. I don't know that translating improves anything in that regard. The call site catch uses a translated message for the toast summary, and then passes this through untranslated as the detailed message. I guess we have other examples of marking highly technical strings for translation, but I'm not sure asking a translator to translate "Missing csrftoken" provides value. It's also a tricky to implement since we're outside a vue component.

Reply:
no that's fair 👍